controlpanel/frontend/jinja2/reset.html

```diff
@@ -61,7 +61,10 @@ <h1 class="govuk-heading-l">Home Directory Reset</h1>
               </div>
             </div>
           </fieldset>
-          <input type="submit" class="govuk-button govuk-button--warning" value="Reset"/>
+          <input type="submit"
+                 class="govuk-button govuk-button--warning cpanel-button--destructive js-confirm"
+                 data-confirm-message="I understand that my conda environment will be irretrievably reset"
```

Comment:
I would probably not mention conda directly (~don't think we do in the rest of the page~ actually we mention as an example of the things that get reset) as we're keeping this "reset your home" vague by design.

Bear in mind the [`user-home-reset` helm chart/job](https://github.com/ministryofjustice/analytics-platform-helm-charts/blob/main/charts/reset-user-home/templates/job.yaml#L21-L27) also wipe other things, e.g. some RStudio stuff - and we may also expand that list as well if new and exciting problems arise in the future.

Maybe tweak to something like:

> I understand that my user home will be reset

Don't know if the ticket mentioned any copy/wording or @ntoll has opinions on this.